xarray/plot/facetgrid.py

```diff
@@ -471,6 +471,74 @@ def add_quiverkey(self, u, v, **kwargs):
         # self._adjust_fig_for_guide(self.quiverkey.text)
         return self
 
+    def _get_largest_lims(self) -> dict[str, tuple[float, float]]:
+        """
+        Get largest limits in the facetgrid.
+
+        Returns
+        -------
+        lims_largest : dict[str, tuple[int | float, int | float]]
+            Dictionary with the largest limits along each axis.
+
+        Examples
+        --------
+        >>> ds = xr.tutorial.scatter_example_dataset(seed=42)
+        >>> fg = ds.plot.scatter("A", "B", hue="y", row="x", col="w")
+        >>> round(fg._get_largest_lims()["x"][0], 3)
+        -0.334
+        """
+        lims_largest: dict[str, tuple[float, float]] = dict(
+            x=(np.inf, -np.inf), y=(np.inf, -np.inf), z=(np.inf, -np.inf)
+        )
+        for k in ("x", "y", "z"):
+            # Find the plot with the largest xlim values:
+            l0, l1 = lims_largest[k]
+            for ax in self.axes.flat:
+                get_lim: None | Callable[[], tuple[float, float]] = getattr(
+                    ax, f"get_{k}lim", None
+                )
+                if get_lim:
+                    l0_new, l1_new = get_lim()
+                    l0, l1 = (min(l0, l0_new), max(l1, l1_new))
+            lims_largest[k] = (l0, l1)
```

Comment:
same here, although less important:
```suggestion
        for axis in ("x", "y", "z"):
            # Find the plot with the largest xlim values:
            lower, upper = lims_largest[axis]
            for ax in self.axes.flat:
                get_lim: None | Callable[[], tuple[float, float]] = getattr(
                    ax, f"get_{axis}lim", None
                )
                if get_lim:
                    lower_new, upper_new = get_lim()
                    lower, upper = (min(lower, lower_new), max(upper, upper_new))
            lims_largest[axis] = (lower, upper)
```

Reply:
Done.